docs/source/upgrade-guide/index.md

```diff
@@ -147,23 +177,48 @@ As a last resort, you can install `react-portal` as a dependency of your project
 However, this is discouraged, because the React 18 rendering could have unexpected side effects.
 It is recommended that you use the React API instead.
 
-### ESlint project configuration update
+### `@plone/registry` moved to ESM
 
 `@plone/registry` and [other packages on which Volto depends](https://github.com/plone/volto/tree/main/packages) are now stand-alone releases in the monorepo structure released in 18.0.0-alpha.4.
 
-You must update the configuration file {file}`.eslintrc.js` in your projects, according to the following `git diff` patch.
+Also, the `@plone/registry` package has been moved to ESM.
+The add-on registry scripts have also been refactored to TypeScript.
+For maximum compatibility with CommonJS builds, the default exports have been moved to named exports.
+The modules affected are now built, and the import paths have changed, too.
+These changes force some import path changes that you should patch in your Plone project or add-on boilerplates.
+
+```{note}
+As always, when something changes in the boilerplate, you may regenerate one from Cookieplone and move your code into it, instead of fiddling with it.
```

Comment:
```suggestion
As always, when something changes in the boilerplate project generator, you may generate a new project with Cookieplone and move your code into it, instead of manually editing multiple files.
See {ref}`upgrade-18-cookieplone-label` for details.
```